### ISSUE TITLE
highlight.js downloads highlighting for all programming languages

### ISSUE THREAD

**Issue by sealer3** [wmitsuda/otterscan#555]
Otterscan has to make hundreds of requests to download all the `chunk-` highlight.js files, which contain the keywords for all the different programming languages. This takes a long time on slow connections. For example, we don't need highlighting for AppleScript or BASIC.

Can we limit this downloading to just Solidity, Vyper, and other Ethereum languages?

**Comment by wmitsuda** [comment]
are you running it from the sources with `npm start`? if so, it's an unoptimized way to run it, I don't know how to solve it (ideas are welcome).

if you are running the production build (either from the docker image or the `npm run build` result) it should do the tree shaking and embed only the solidity/json highlighters.

**Comment by sealer3** [comment]
You are right, I was running with `npm start` :) I was looking to run Otterscan without the use of Docker, and it appears I overlooked that it was a development version and not for production use. I was following this guide in the docs: [docs/other-ways-to-run-otterscan.md](https://github.com/wmitsuda/otterscan/blob/develop/docs/other-ways-to-run-otterscan.md) It would be useful to include instructions on how to run the production version in a portable way without Docker. I can create these instructions once I find a nice way of doing it.

I got it to run with `npm run build` to get all of the production assets into the `dist` folder, I customized the `config.json` to point to the correct URLs, and ran

```bash
python3 -m http.server 5173 --bind localhost --directory ./dist
```
to create a simple HTTP server for now. (Of course, a real solution would serve `index.html` to more paths so direct requests to pages are handled correctly.)
